app/components/app/index.js

```diff
@@ -1,8 +1,10 @@
 import AppComponent from './app.component';
+import AppService from './app.service';
 
 const appModule = angular
     .module('app.components.appModule', [])
     .component('appComponent', AppComponent)
+    .service('appService', AppService)
```

Comment:
Not sure if `appService` is good name - it doesn't tell us anything what that service will do